index.html

```diff
@@ -13,10 +13,22 @@
                 background-color:#30323d;
             }
 
-            p {
+            p.blocktext {
                 font-family: 'Open Sans', sans-serif;
                 color:#cdd1c4;
                 font-size: 17px;
+                margin-left: auto;
+                margin-right: auto;
+                width: 27em
```

Comment:
There also needs to be a semi-colon after all of the key value pairs or the change won't be seen.